### ISSUE TITLE
[21248] Phrasing of DD Talk/Chat within the Customer Portal

### ISSUE THREAD

**Issue by Aycorn** [DeskDirector/UserVoice#403]
Currently, the following phrasing is static:

"Chat to a Technician
Get the attention of a Technician for immediate support, or just to ask a quick question."

![image](https://user-images.githubusercontent.com/30588784/70749777-22cbf500-1d81-11ea-942e-c020da845351.png)

Feature request to have this phrasing adjustable as a setting within the Admin Console.

The use case for this is having your dispatch team answering chat messages, and avoiding a false promise of connecting them to a Technician.

**Comment by Lowell-Syncrety** [comment]
Yes, this would be awesome. We're thinking of rolling out chat even though we don't have the resources to support it, but would need specific text that would let folks know that no one is typically available and if someone can't answer their question right away it will convert into a ticket for them.

**Comment by noahbalboah** [comment]
Yes please. We'd like to adjust the wording so we can use chat more as a dispatching feature than to get immediate support.
